filters/f_decoder_wrapper.c

```diff
@@ -555,15 +554,27 @@ static void fix_image_params(struct priv *p,
         MP_VERBOSE(p, "Decoder format: %s\n", mp_image_params_to_str(params));
     p->dec_format = *params;
 
+    if (!quiet && (opts->movie_aspect == 0 || opts->movie_aspect == -1))
+        MP_WARN(p, "Setting video-aspect-override to 0 or -1 is deprecated.\n"
+                   "Use video-aspect-mode instead.\n");
```

Comment:
I think it's probably best to split this up into two different warnings because getting the behavior with `0` requires `--video-aspect-method` whereas `-1` is still `--video-aspect-override`.